rfcs/text/0000-tech-planning-process.md

```diff
@@ -0,0 +1,136 @@
+
+**Note: MXWY is equal to “week Y in milestone X.”
+
+**Note: this schedule may differ based on the holiday schedule each quarter. The responsibility is on the Shepherd of
+the quarter to make sure the timeline is clear (especially the date of the TL+PM call) prior to the process commencing.
+
+### Templates
+
+For the purpose of curating the individual component objectives and for the consolidation of objectives into one that
+represents all the components’ objectives, please use: 
+[TL Objectives Template](https://docs.google.com/document/d/1jUo9Y7y9BJhVe8qpgljI45W7JpNlq3ownVBLA3wK0Qw/edit).
+
+Once the consolidation has taken place, as well as the TL meeting in Week 9, the TL Roadmapping Shepherd MUST turn the
+document into an Informational RFC and publish. The template for crafting an RFC for this purpose is 
+[here](https://github.com/HumanCellAtlas/dcp-community/blob/4bb18c65af5b244460cf404465b1dbc4ee0cc65e/rfcs/roadmap-template.md) 
+but should be tailored such that Themes are removed and replaced with the technical priorities.
+
+### Specific Shepherd Tasks
+
+1) Find out the date/week of the TL+PM planning call and post a timeline on when artifacts are expected to be completed
+to Slack **#tech-architecture** so that folks are aware that the planning process has begun and expectations are set.
+
+2) Schedule a 1 hour TL Coalescing Meeting and a 1 hour TL Prioritization Meeting, including all TLs\*, one and two
+weeks after the individual components have been assigned to start working on their individual components’ objectives,
+respectively.
+
+3) If the meetings aren't separately scheduled, by default it will occur during the two consecutive Tech Architecture
+meetings starting one week after components have been told to begin work on enumerating their objectives.
+
+4) Transform the document produced at the TL Prioritization meeting into an RFC and publish.
+
+5) **Annoyingly** send reminders about the timeline and expected artifacts and communicate consistently and frequently.
+*If you aren’t annoying, you didn’t send out enough reminders.*
+
+6) Shepherd along the TL Coalescing/Prioritizing meetings such that we get to a final set of priorities. Also act as the
+primary representative of the TLs during the TL+PM meeting when needed.
+
+\* If a TL is unable to attend the meeting due to unavoidable conflicts, a representative may attend instead.
+
+### Unresolved Questions
+
+- What do we do if the timeline slips and the TLs are not able to produce an artifact on time for the TL+PM call?
+  - One suggested consequence: the roadmaps+planning RFC defines the outcome regardless of whether TL is contribute on
+  time or not.
+    
+- How do we hold each component accountable to make sure they contribute to the planning process? The minimum
```

Comment:
All you can do is exert social pressure. If a component doesn't contribute then they don't get their voice heard, that is all.

Reply:
Makes sense to me.